.github/workflows/ci_windows_x64_msvc.yml

```diff
@@ -22,19 +22,22 @@ concurrency:
 
 jobs:
   windows_x64_msvc:
-    runs-on: windows-2022
-    defaults:
-      run:
-        shell: bash
+    runs-on: arc-runner-set
     env:
-      BUILD_DIR: build-windows
+      BUILD_DIR: C:\mnt\azure\build-windows
     steps:
+      - name: "Install choco reqs"
+        run: |
+          Get-PSDrive | Format-Table Name, Used, Free
+          Test-Path C:\mnt\azure
+          Get-ChildItem -Path C:\mnt\azure
+          fsutil volume diskfree C:\mnt\azure
```

Comment:
All generic (non-project-specific) setup should be moved to the Dockerfile. I'm less concerned about steps taking up to 1 minute, but the 15 minute MSVC install will need to be moved to ahead of time. Our target for presubmit should be 10-20 minutes per run, including the source build (warm cache). We can budget for 5 minutes or so of overhead (git checkout, installing deps, etc.).